src/interactive/Tooltip.jsx

```diff
@@ -60,6 +60,10 @@ class Tooltip extends React.PureComponent {
 		}
 	}
 
+	handleClick = () => {
+		this.setState(prevState => ({ isActive: !prevState.isActive }));
```

Comment:
nitpick, but avoid calling the arg to the updater `prevState`, which is a bit misleading (I actually made the update to the React docs in [this PR](https://github.com/reactjs/reactjs.org/pull/1155))